### PR TITLE
Fixes issue #180

### PR DESCRIPTION
**Pull request by eaescob**
find_oper() logic now checks for nick before iterating through host list and takes nick match into consideration when deciding to return aoper pointer when CIDR matches.

### CODE REVIEW

File: src/s_conf.c [hunk 1]
```diff
@@ -394,42 +395,46 @@ find_oper(char *name, char *username, char *sockhost, char *hostip)
     {
         if (aoper->legal == -1)
             continue;
+        if(!mycmp(name, aoper->nick))
+            nickmatch=1;
+        else
+            continue;
```
Comment (crigler) [inline src/s_conf.c:401]: It looks like `nickmatch` will always be 1 after this point. Is the `nickmatch` variable necessary?

**Review by crigler — CHANGES_REQUESTED** [review]

File: src/s_conf.c (hunk 1 above)
Comment (kobishmueli) [reply src/s_conf.c:401]: Good point, I would just use:
if(mycmp(name, aoper->nick))
    continue;

As there is no point to waste CPU for checking a CIDR for an entry that isn't the one we're looking for anyway...

Kobi.

Comment (eaescob) [reply src/s_conf.c:401]: Damn. Good points. Thanks

**Review by crigler — DISMISSED** [review]
Is there a way to update the pull request to squash all the commits together and remove the whitespace changes? Other than that it looks good now.

**Comment by eaescob** [comment]
There, I think this looks better. Ironically, my GH desktop app doesn't show the other space/tab conversion changes so wonder if GH web is doing that.

**Pull request merged**